cfme/tests/automate/test_service_dialog.py

```diff
@@ -481,3 +482,57 @@ def test_automate_methods_from_dynamic_dialog_should_run_as_per_designed(
         for ele_name in [ele_label, "label2", "label3"]:
             # Checking if automate method is not triggered after updating values of dialog widgets
             view.fields(ele_name).input.fill(fauxfactory.gen_alphanumeric())
+
+
+@pytest.mark.meta(automates=[BZ(1677724)])
+@pytest.mark.customer_scenario
+@pytest.mark.tier(2)
+def test_service_dialogs_crud_non_admin_user(appliance, user_self_service_role):
+    """
+
+    Bugzilla:
+        1677724
+
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Create a user with tenant admin group
+            2. Log in with said user and try to edit/add service dialog
+        expectedResults:
+            1.
+            2.
+            3. User should able to perform crud operations
+    """
+    user, role = user_self_service_role
+
+    # with update(role):
+    product_features = [
```

Comment:
**Question:**
Why do we need line(511) comment?
Please remove if not required.